File: 07-Problem-Variables/07-Problem-Variables.js
// Create a mutable variable and an immutable variable and print each in one log.  Add the immutable variable to the mutable variable. Again print both in a new log.

let mutable = 10;
const immutable = 19;

console.log(`mutable: ${mutable} immutable: ${immutable}`);

mutable += immutable;

console.log(`mutable: ${mutable} immutable: ${immutable}`);

File: 07-Problem-Variables/07_Problem_Variables.py
# Create a mutable variable and an immutable variable and print each in one log.  Add the immutable variable to the mutable variable. Again print both in a new log.

my_mutable = 10
MY_IMMUTABLE = 19

print(f"my_mutable: {my_mutable} my_immutable: {MY_IMMUTABLE}")

my_mutable += MY_IMMUTABLE

print(f"my_mutable: {my_mutable} my_immutable: {MY_IMMUTABLE}")


